homeassistant/components/sensor/wunderground.py

```diff
@@ -29,50 +29,562 @@
 
 DEFAULT_LANG = 'EN'
 
-MIN_TIME_BETWEEN_UPDATES_ALERTS = timedelta(minutes=15)
-MIN_TIME_BETWEEN_UPDATES_OBSERVATION = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
```

Comment:
@pezinek with this change, are you considering the threshold of 500 calls a day?